apps/web/src/app/page.tsx

```
@@ -1,321 +1,665 @@
+"use client";
+import GeneralCTA from "@/components/cta";
+import { Badge } from "@/components/ui/badge";
+import { Button, buttonVariants } from "@/components/ui/button";
+import { Card, CardContent } from "@/components/ui/card";
+import {
+  ArrowRight,
+  Calendar,
+  Camera,
+  Check,
+  Download,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)